data/json/items/containers.json

```diff
@@ -187,6 +187,23 @@
     "symbol": ")",
     "color": "black"
   },
+  {
+    "id": "bag_garbage_reinforced",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "reinforced garbage bag" },
+    "looks_like": "bag_canvas",
```

Comment:
```suggestion
    "looks_like": "bag_garbage",
```

Reply:
i was unaware this worked, i copied that from the original garbage bag entry which should also be updated